src/components/Search/SearchOverTime.tsx

```diff
@@ -17,38 +18,69 @@ import {
   SearchStyles,
 } from '../../styles';
 import { Changeset, Repository } from '../../types/state';
+import { TimeRange } from '../../types/types';
 import SearchDropdown from './SearchDropdown';
 import SearchInputAndResults from './SearchInputAndResults';
 import SelectedRevisions from './SelectedRevisions';
+import TimeRangeDropdown from './TimeRangeDropdown';
 
 interface SearchProps {
-  selectLabel: string;
-  tooltip: string;
-  inputPlaceholder: string;
   hasEditButton: boolean;
   displayedRevisions: Changeset[];
-  repository: Repository['name'];
+  baseRepo: Repository['name'];
+  newRepo: Repository['name'];
+  timeRangeValue: TimeRange['value'];
   onRemoveRevision: (item: Changeset) => void;
   onSearchResultsToggle: (item: Changeset) => void;
-  onRepositoryChange: (repo: Repository['name']) => unknown;
+  onBaseRepositoryChange: (repo: Repository['name']) => unknown;
+  onNewRepositoryChange: (repo: Repository['name']) => unknown;
+  onTimeRangeChange: (val: TimeRange['value']) => unknown;
 }
 
 export default function SearchOverTime({
-  selectLabel,
-  tooltip,
-  inputPlaceholder,
   hasEditButton,
-  repository,
+  baseRepo,
+  newRepo,
   displayedRevisions,
+  timeRangeValue,
   onRemoveRevision,
   onSearchResultsToggle,
-  onRepositoryChange,
+  onBaseRepositoryChange,
+  onNewRepositoryChange,
+  onTimeRangeChange,
 }: SearchProps) {
   const mode = useAppSelector((state) => state.theme.mode);
   const styles = SearchStyles(mode);
+  const timeRangeStrings =
+    Strings.components.searchDefault.overTime.collapsed.timeRange;
+  const stringsBase =
+    Strings.components.searchDefault.overTime.collapsed.baseRepo;
+  const stringsNew =
+    Strings.components.searchDefault.overTime.collapsed.revisions;
 
   const [formIsDisplayed, setFormIsDisplayed] = useState(!hasEditButton);
 
+  const baseRepoStyles = style({
+    marginBottom: `${Spacing.Large}px`,
+    justifyContent: 'space-between',
```

Comment:
I'm surprised this needs to be specified as CSS instead of using properties on the `<Grid>` component.

Reply:
Yes, I agree. I'll use MUI grid properties instead.